query-service-api/src/main/proto/request.proto

```diff
@@ -14,6 +14,7 @@ message Expression {
     LiteralConstant literal = 2;
     Function function = 3;
     OrderByExpression orderBy = 4;
+    ObjectIdentifier objectIdentifier = 5;
```

Comment:
This is against the lint rules, but I guess consistency is better.

Reply:
resolved